tools/ld-analyse/CMakeLists.txt

```diff
@@ -0,0 +1,246 @@
+message( STATUS "---------Beginning cmake build of ld-analyse---------")
+
+# Qt Widgets for Technical Applications
```

Comment:
Put this in `cmake/FindQwt.cmake` at the top level - don't copy the text in yourself.